herd/libdir/aarch64hwreqs.cat

```diff
@@ -123,18 +132,29 @@ let pick-lob = pick-basic-dep; lob; [W]
 
 let TLBUncacheable = MMU & (Translation | AccessFlag)
 
-(* Hardware-required-ordered-before *)
-let rec hw-reqs =
+(* Locally-hardware-required-ordered-before *)
+let rec local-hw-reqs =
   tr-ib
   | tc-ib
   | [Exp & M | Imp & T & R]; (lob | pick-lob); [Exp & M | Imp & T & R | FAULT & (TagCheck | MMU)]
   | (if "ETS2" then [Exp & M]; po; [TLBUncacheable & FAULT]; tr-ib^-1; [Imp & TTD & R] else 0)
   | DSB-ob
   | CSE-ob
-  | [CSE]; po
   | [Imp & TTD & R]; po-loc; [W]
   | [Imp & TTD & R]; rmw; [HU]
   | [Exp & M]; po-loc; [TLBUncacheable & FAULT]
-  | [Exp & R]; addr; [TLBI]
+  | [Exp & R]; addr; [TLBI | DC.CVAU | IC]
   | [Exp & R]; ctrl; [HU]
-  | hw-reqs; hw-reqs
+
+  | f-ib
+  | [Imp & INSTR & R]; po; [~(Imp & INSTR & R)]
+
+  | [DC.CVAU]; po; [dmb.full]; po; [DC.CVAU]
```

Comment:
please reorder so the enumerations match each other

Reply:
Would the following implement your suggestion?
```
  | [M & Exp]; po; [dmb.full]; po; [DC.CVAU]
  | [DC.CVAU]; po; [dmb.full]; po; [M & Exp]
  | [DC.CVAU]; po; [dmb.full]; po; [DC.CVAU]

  | [Exp & M]; (po & scl); [DC.CVAU]
  | [DC.CVAU]; (po & scl); [M & Exp]
  | [DC.CVAU]; (po & scl); [DC.CVAU]
```